### PR TITLE
Pages/Chapters for Form 4142 Release of Medical Records

### PR DESCRIPTION
**Pull request by rhasselle-oddball**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
- Add initial pages and chapters
- Expect ongoing feedback
- Initial code was done for Provider facilities (array of items), but full solution will be taken care of in another ticket

## Related issue(s)
- department-of-veterans-affairs/va.gov-team-forms#90

## Testing done

- UI field testing

## Screenshots
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

## What areas of the site does it impact?
new route `/health-care/medical-release-form/`

## Acceptance criteria
- [x]  I added a screenshot of the developed feature

## Requested Feedback
- Would like feedback on titles.
- Not necessarily looking for feedback on provider facilities page, since that will be handled in another ticket

### CODE REVIEW

File: src/applications/simple-forms/21-4142/pages/authorization.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          <p>
+            I understand that VA may use information disclosed prior to
+            revocation to decide my claim.
+          </p>
+        </>
+      ),
+    },
+    acknowledgeToReleaseInformation: {
+      'ui:title': 'I acknowledge and authorize this release of information',
+      'ui:required': () => true,
```
Comment (ndsprinkle) [inline src/applications/simple-forms/21-4142/pages/authorization.js:102]: Fairly minor, but I don't think you need this line because you also put `acknowledgeToReleaseInformation` in the `required` array in the schema below.

File: src/applications/simple-forms/21-4142/pages/authorization.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  schema: {
+    type: 'object',
+    required: ['acknowledgeToReleaseInformation'],
+    properties: {
+      'view:legalText': {
+        type: 'object',
+        properties: {},
+      },
+      acknowledgeToReleaseInformation:
+        fullSchema.properties.acknowledgeToReleaseInformation,
```
Comment (ndsprinkle) [inline src/applications/simple-forms/21-4142/pages/authorization.js:114]: Also fairly minor, but if you wanted to follow the pattern from the other pages, you could use your constants definition here for `acknowledgeToReleaseInformation`. The limitations page you added looks very similar and uses the constants in the schema like this: `fullSchema.properties[schemaFields.limitedConsent],`

File: src/applications/simple-forms/21-4142/pages/limitations.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          <h3 className="vads-u-color--gray-dark vads-u-margin-top--0">
+            Do you want to limit the information we can request?
+          </h3>
+          <p>
+            If you want to limit what we can request from your private medical
+            provider(s), describe the limits here (for example, you want your
+            doctor to release only treatment dates or certain types of
+            disability), it may take us longer to get your private medical
+            recordsif you limit consent.
+          </p>
```
Comment (ndsprinkle) [inline src/applications/simple-forms/21-4142/pages/limitations.js:19]: Does this paragraph section make more sense to be inside `ui:description`? I don't know how it affects the UI, but it may make more accessibility sense if the title is just the `h3` part and the description is the rest. This is also fairly minor, like my other comments

File: src/applications/simple-forms/21-4142/pages/personalInformation2.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+];
+const pageFields = [veteranFields.ssn, veteranFields.vaFileNumber];
+
+export default {
+  uiSchema: {
+    [veteranFields.parentObject]: {
+      'ui:title': (
+        <h3 className="vads-u-color--gray-dark vads-u-margin-top--0">
+          Identification numbers
+        </h3>
```
Comment (ndsprinkle) [inline src/applications/simple-forms/21-4142/pages/personalInformation2.js:18]: I know we had a sync about this, but I noticed my sketch has indications like "use an h3, but apply h2 styling", so that may be the answer to some of the questions we had. So maybe some of these need to change to h2 stylings? Might be worth shooting a message to Jeana about, but I know there are several locations on the pages that would need to be update

image

File: src/applications/simple-forms/21-4142/pages/recordsRequested.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+      [providerFacilityFields.parentObject]: {
+        type: 'array',
+        items: {
+          ...fullSchema.properties[providerFacilityFields.parentObject].items,
+          properties: {
+            ...fullSchema.properties[providerFacilityFields.parentObject].items
+              .properties,
+            [providerFacilityFields.providerFacilityAddress]: address.schema(
+              fullSchema,
+              () => true,
```
Comment (ndsprinkle) [inline src/applications/simple-forms/21-4142/pages/recordsRequested.js:64]: The schema kinda looks funny, but I guess that's because we have to overwrite the address schema? We'll have to figure that out at some point. The issue is misalignment between the standard address definition in vets-json-schema and the address definition in the forms-library.

File: src/applications/simple-forms/21-4142/pages/recordsRequested.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+            ),
+          },
+        },
+        default: [
```
Comment (ndsprinkle) [inline src/applications/simple-forms/21-4142/pages/recordsRequested.js:68]: Just curious, why are these necessary? Is it related to array functionality?

**Review by ndsprinkle — APPROVED** [review]

**Review by pjholyoke — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM
